gui/src/app/login/login.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container-fluid pt-5 p-lg-5">
+  <h4 class="card-title">
+    {{'LOGIN_MESSAGE' | translate}}
+  </h4>
+  <button class="ml-2" mat-raised-button color="primary" (click)="login()">
```

Comment:
I would change the color to `accent`. Since the last color changes, I would like not to have any more buttons with the primary (black) color.